odxtools/compumethods/makeanycompumethod.py

```diff
@@ -79,37 +77,10 @@ def _parse_compu_scale_to_linear_compu_method(scale_element,
 
     return LinearCompuMethod(offset=offset, factor=factor, **kwargs)
 
-
-def read_limit_from_odx(et_element, internal_type: DataType):
-    limit: Optional[Limit] = None
-    if et_element is not None:
-        if et_element.get("INTERVAL-TYPE"):
-            interval_type = IntervalType(et_element.get("INTERVAL-TYPE"))
-        else:
-            interval_type = IntervalType.CLOSED
-
-        if interval_type == IntervalType.INFINITE:
-            if et_element.tag == "LOWER-LIMIT":
-                limit = Limit(float("-inf"), interval_type)
-            else:
-                assert et_element.tag == "UPPER-LIMIT"
-                limit = Limit(float("inf"), interval_type)
-        else:
-            if internal_type == DataType.A_BYTEFIELD:
-                hex_text = et_element.text
-                if len(hex_text) % 2 == 1:
-                    hex_text = '0' + hex_text
-                limit = Limit(bytearray.fromhex(hex_text), interval_type)
-            else:
-                limit = Limit(internal_type.from_string(et_element.text),
-                              interval_type)
-    return limit
-
-
-def read_compu_method_from_odx(et_element,
-                               doc_frags: List[OdxDocFragment],
-                               internal_type: DataType,
-                               physical_type: DataType) -> CompuMethod:
+def make_any_compu_method_from_et(et_element,
```

Comment:
"any" sounds redundant, even without it the function role is pretty clear.
```suggestion
def make_compu_method_from_et(et_element,
```

Reply:
hm, I want to somehow signify that this is a factory method...